ansible/roles/invoker/tasks/deploy.yml

```diff
@@ -262,12 +262,17 @@
 
 - name: set invoker volumes
   set_fact:
-    volumes: "/sys/fs/cgroup:/sys/fs/cgroup,/run/runc:/run/runc,\
+    volumes: "/sys/fs/cgroup:/sys/fs/cgroup,\
       {{ whisk_logs_dir }}/{{ invoker_name }}:/logs,\
       {{ invoker.confdir }}/{{ invoker_name }}:/conf,\
       {{ dockerInfo['DockerRootDir'] }}/containers/:/containers,\
       {{ docker_sock | default('/var/run/docker.sock') }}:/var/run/docker.sock"
 
+- name: set invoker runc volume
+  set_fact:
+    volumes: "{{ volumes }},/run/runc:/run/runc"
+  when: invoker_use_runc == true
```

Comment:
`invoker_use_runc`is not reachable here. 
it probably should be `when: invoker.useRunc == true`

Reply:
Thanks. Updated.